Smash_Combos.Domain/Models/Report.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Smash_Combos.Domain.Models
+{
+    public class Report
```

Comment:
Report should have `public User User` and `public User Reporter` rather than the ids